flask_dynamo/manager.py

```diff
@@ -148,6 +149,40 @@ def _get_ctx(app):
                 'flask-dynamo extension not registered on flask app'
             )
 
+    @staticmethod
+    def _init_session(app):
+        session_kwargs = {}
+        # Only apply if manually specified: otherwise, we'll let boto
+        # figure it out (boto will sniff for ec2 instance profile
+        # credentials).
+        if app.config['AWS_ACCESS_KEY_ID']:
```

Comment:
https://github.com/rdegges/flask-dynamo/pull/28/commits/7ba530b120fab47abf2bc8537182e07cc29f2ba1 addresses these concerns. Thanks for the feedback!